### DISCUSSION TITLE
How to fully uninstall?

### DISCUSSION THREAD

**Discussion by petezhut** [X3n0m0rph59/eruption#194] [Q&A]
After trying out eruption for a few months, I have not managed to do anything with it except fill up my drive with repeated:
 

> Dec 23 21:36:32 jester Jester eruption[888]: An error occurred while closing the device  
> Dec 23 21:36:32 jester Jester eruption[888]: Could not process a mouse event: receiving on a closed channel  
> Dec 23 21:36:32 jester Jester eruption[888]: An error occurred while closing the device  
> Dec 23 21:36:32 jester Jester eruption[888]: Could not process a mouse event: receiving on a closed channel  
> Dec 23 21:36:32 jester Jester eruption[888]: An error occurred while closing the device  
> Dec 23 21:36:32 jester Jester eruption[888]: Could not process a mouse event: receiving on a closed channel  
> Dec 23 21:36:32 jester Jester eruption[888]: An error occurred while closing the device  

messages in /var/log/syslog to the tune of >100G/day.  I am trying to figure out how to fully remove this from my Ubuntu system.  I have tried apt remove and turning off services but all that does is cause my system to crash trying to load a missing module for eruption.  Does anyone have a step-by-step guide to remove this from a Ubuntu system?  Thank you for your assistance in this matter.

**Comment by X3n0m0rph59** [comment]
Hi @petezhut, you can uninstall Eruption by running:

```sh
sudo make uninstall
````

from the directory to where you cloned the project to.

If you used a package manager instead of building from source then please use the package manager to uninstall

**Reply by petezhut** [reply]
So, I ended up having to reinstall my OS to fully remove it.  It was leaving behind something that the kernel kept trying to load.  All the best.
